### DISCUSSION TITLE
How do I turn lights on and off in RedCube.js?

### DISCUSSION THREAD

**Discussion by cx20** [Reon90/redcube#21] [Q&A] [answered]
I would like to add a GUI to the `RedCube.js` sample in `gltf-test` that can turn lights on and off like the `Babylon.js` sample.
Below is an image of the Babylon.js sample GUI.

![image](https://user-images.githubusercontent.com/1833979/106208371-97bcf100-6206-11eb-9309-d8519e3f64fb.png)

However, I don't know how to use RedCube.js and I don't know how to turn the light on and off. I would appreciate it if you could teach me how to use it.

If it can be realized, I believe that the display result similar to Babylon.js will be obtained.

|Babylonj.js + Lights off|RedCube.js|
|:----:|:----:|
|![image](https://user-images.githubusercontent.com/1833979/106208151-46acfd00-6206-11eb-9448-bef26fcb1a2d.png)|![image](https://user-images.githubusercontent.com/1833979/106208212-617f7180-6206-11eb-8e99-bab4cefc327b.png)|

**Comment by Reon90** [comment, marked as answer]
@cx20 it's possible, please call:
```
redcube.isIBL = false;
redcube.isDefaultLight = false;
redcube.draw();
```

**Reply by cx20** [reply]
@Reon90 Thank you for teaching me how to use it.
I would like to have glTF built-in lights illuminated even if both `IBL` / `Lights` are `off`. is it possible?

||IBL off|IBL on|
|:--:|:--:|:--:|
|Lights off|![image](https://user-images.githubusercontent.com/1833979/108000801-24044c00-702e-11eb-9c0c-260d4ba079b2.png)|![image](https://user-images.githubusercontent.com/1833979/108000810-28c90000-702e-11eb-97af-dd37c24d6ea9.png)|
|Lignts on|![image](https://user-images.githubusercontent.com/1833979/108000830-2ebee100-702e-11eb-80a4-1e72655f276d.png)|![image](https://user-images.githubusercontent.com/1833979/108000836-341c2b80-702e-11eb-8c0d-768489968c6e.png)|

**Reply by Reon90** [reply]
@cx20 Fixed in 2.5.6

**Reply by cx20** [reply]
@Reon90 Thanks! I tried it with v2.5.6. It looks good.

||IBL off|IBL on|
|:--:|:--:|:--:|
|Lights off|![image](https://user-images.githubusercontent.com/1833979/108063866-34084400-709f-11eb-9196-ee937f307799.png)|![image](https://user-images.githubusercontent.com/1833979/108063879-38ccf800-709f-11eb-9f5c-9293a4c6c522.png)|
|Lignts on|![image](https://user-images.githubusercontent.com/1833979/108063891-3ec2d900-709f-11eb-8530-56c15af3a742.png)|![image](https://user-images.githubusercontent.com/1833979/108063908-44b8ba00-709f-11eb-8f18-7bd28a1d8e8d.png)|
